### ISSUE TITLE
[Suggestion] Add toString() method for listAddress()

### ISSUE THREAD

**Issue by cloud8little** [neow3j/neow3j#3]
current the built-in method toString() print the following: 
[io.neow3j.protocol.core.methods.response.NeoListAddress$Address@7fe8ea47, io.neow3j.protocol.core.methods.response.NeoListAddress$Address@226a82c4]
Suggest add override toString method similar with below:
```
        NeoListAddress neoListAddress = (NeoListAddress)neow3j.listAddress().send();
        List<NeoListAddress.Address> address= neoListAddress.getAddresses();
        System.out.println(address.size());
        for (int i = 0; i < address.size(); i++){
            System.out.println(address.get(i).getHasKey() + ":" + address.get(i).getAddress() + ":" + address.get(i).getLabel() + ":" + address.get(i).getWatchOnly());
        }
```

**Comment by gsmachado** [comment]
Thanks for the suggestion, @cloud8little.

However, the `NeoListAddress.getAddresses()` returns a `List` -- which is a Java type. It's not advisable to override the `.toString()` in this case.

In order to print the variable `address` in this case, why don't you do, for example:

```
System.out.println(Arrays.toString(address.toArray()));
```

**Comment by gsmachado** [comment]
@cloud8little a fix was released on `1.0.7`.
If you experience that the issue was not yet fixed, please re-open it.

**Comment by cloud8little** [comment]
Verified on 1.0.7, it will return responses as required.
`[Address{address='AcDZPbtcK8djLk5ZXLkps9XL7ee45RHkBB', hasKey=true, label='null', watchOnly=false}, Address{address='AMnXC3U1PxdDqDNHbeL23ZfAfGVL9d7yYz', hasKey=true, label='null', watchOnly=false}, Address{address='AdfNNP8Fw3cbZWcYynig6MtwoKrxdyqMHs', hasKey=true, label='null', watchOnly=false}, Address{address='AUdBXmmjACFtxEUT1zsAvMSC5WnuY7oeU1', hasKey=true, label='null', watchOnly=false}, Address{address='AR17sBSuDxesg5BDUT6UjkH2Rs5KxCtHtW', hasKey=true, label='null', watchOnly=false}, Address{address='AViGHdsrDS1JnEADxE6cWdi2VFc3RWVgVA', hasKey=true, label='null', watchOnly=false}]`
